main/templates/main/index.html

```diff
@@ -0,0 +1,430 @@
+            <ul id="menu">
+              <a href="#"><li><i class="fa fa-home" aria-hidden="true" style="font-size:24px;"></i>&nbsp;&nbsp; Home</li></a>
+              <a href="#"><li><i class="fa fa-id-card" aria-hidden="true" style="font-size:24px;"></i>&nbsp; Contact</li></a>
+              <a href="#"><li><i class="fa fa-user-plus" aria-hidden="true" style="font-size:24px;"></i>&nbsp; Register</li></a>
+                <a href="#"><li><i class="fa fa-sign-in" aria-hidden="true" style="font-size:24px;"></i>&nbsp;&nbsp; Log In</li></a>
+            </ul>
+          </div>
+        </nav>
+        
+		<div class="bgimg-1">
+		  <div class="header" id="nav">
+			  <a href="#index" class="logo" style="font-family: 'Kaushan Script', cursive;">
+                  <span style="font-weight: normal; font-family: 'Kaushan Script', cursive;">Lets</span> Meet
+              </a>
+			  <div class="header-right">
+				<a class="active" href="#home">Home</a>
+				<a href="#second">Contact</a>
+				<a href="#footer">About</a>
+                <a href="#log in">Log In</a>
+                <a href="#register">Register</a>
+			  </div>
+		  </div>
+		  <div class="caption">
+              <span class="text" style="font-family: 'Kaushan Script', cursive;">Lets Meet</span>
+		  </div>
+		</div>
+
+        <div class="bgimg-1 drop-shadow">
+            <div class="glass"></div>
+            <span id="glassText">
+              There is a difficulty coordinating meetings between different people, whether it’s a meeting between teammates, <br>
+                meeting between a few students, catching up with friends or family and more, due to a busy schedule <br>
+                that varies between person to person. <br>
+                Therefore, in order to solve this problem we would like to develop a platform named “Lets Meet”. <br>
+                The platform is meant to help the users to manage meetings of different types <br>
+                and centralize related information that links to those meetings such as files, chat and links to the meetings. <br>
+                Our site is meant to be used by everyone. 
+            </span>
+        </div>
+
+
+        <div class="bgimg-2">
+		  <div class="caption">
+			<span class="border" id="wallpaperCaption" style="color: #f7f7f7;">What We Offer</span>
+		  </div>
+		</div>
+  
+        <div class="leftBoxText">
+            <div class="row" style="color: white">
+              <div class="column columnBox">
+                  <h2 style="font-size: 4em;" onclick="eventCreation()"><u>Set Events</u></h2>
+                <p style="font-size: 3em;">
+                    Events are the user way to create an action with that will happen <br> 
+                    in the future, or that happend in the past and wanted to be rememberd. <br>
+                    An Event is composed of a title, data and time, location and a description. <br>
+                    Every event is linked to the user who created it.
```

Comment:
maybe need to add a margin on the left side

Reply:
I have changed the order of the words so now it will not stick to the side